week03/design_hashmap.cpp

```diff
@@ -0,0 +1,53 @@
+// TC: O(1)
+// MC: O(n) where n is the number of operations
+
+class MyHashMap
+{
+public:
+    const static int MAXSIZE = 103880;
```

Comment:
too large.

Reply:
Vai, will it be a problem in interview round?